include/led-matrix.h

```diff
@@ -103,6 +104,9 @@ class RGBMatrix : public Canvas {
   // animation.
   FrameCanvas *SwapOnVSync(FrameCanvas *other);
 
+  void SetTransformer(CanvasTransformer *transformer);
+  CanvasTransformer *transformer();
```

Comment:
You could make that an inline implementation here.

Also we should make sure that whatever we return here is never NULL to not put the burden of NULL checking to the enduser. We should just set transformer_ to an IdentityTransformer when SetTransformer() passes in a NULL.